drivers/i2c/Kconfig.stm32

```diff
@@ -65,4 +65,13 @@ config I2C_STM32_V2_DMA
 	help
 	  Enable DMA support for the STM32 I2C driver.
 
+config I2C_STM32_TRANSFER_TIMEOUT_MSEC
+	int "STM32 I2C transfer timeout (ms)"
+	depends on I2C_STM32
```

Comment:
You can remove this line. This config definition is already conditioned between `#if I2C_STM32` (line 12) and `#endif` (line 77).